spec/fixtures/ial2_test_credential_with_face_match_fail.yml

```diff
@@ -0,0 +1,18 @@
+portrait_match_results:
```

Comment:
it looks like these fixtures are not referenced anywhere? is that intentional?

Reply:
Thanks for bringing that up, I meant to ask about that. I have been using these fixtures as a way to test locally, but I learned from @soniaconnolly that perhaps I should be using https://developers.login.gov/testing/#data-testing instead (I believe that was the location you mentioned Sonia, please correct me if I'm wrong).

So I added these as a way to have them handy for people like myself that use them for local testing, but maybe that was the wrong choice. Are the fixtures here only meant for spec files to run? If so, I'd agree they should probably be removed.

(It's also a good reminder to update those docs, as one includes a `liveness_result` that would not be compatible with these changes (or before them).)

Reply:
The doc for testing yml is [here](https://github.com/18F/identity-dev-docs/blob/b4f81d2f687a4d11da74ad9a26a7de11def0c0ff/_pages/testing.md?plain=1#L2).

Reply:
Oh yep! That looks like the code for the page I linked.

Reply:
Removed the fixture files in [0861a4d](https://github.com/18F/identity-idp/pull/9882/commits/0861a4d138333238e809bffa5b29af7df53b497d).

Updated the docs in https://github.com/18F/identity-dev-docs/pull/425.